test.py

```diff
@@ -84,6 +84,75 @@ def test_divisions_becomes_present(edap):
     assert not edap.org_unit_exists("divisions")
 
 
+def test_special_becomes_present(edap):
+    assert not edap.org_unit_exists("special")
+    edap.create_org_unit("special")
+    assert edap.org_unit_exists("special")
+
+
+def test_presidium_special_becomes_present(edap):
+    description = b'presidium'
+    presidium = "presidium"
+    special = "special"
+    # print("test_presidium_special_becomes_present begin:", f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup")
+    # print("edap.object_exists_at", f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup", "=", edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup"))
+    assert not edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup")
+    edap.create_special(presidium, display_name=presidium)
+    # print("edap.object_exists_at", f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup", "=", edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup"))
+    assert edap.object_exists_at(f"cn={presidium},{edap.SPECIAL_GROUP}", "posixGroup")
+    res = edap.get_special(presidium)
+    # print("edap.get_special(presidium)", res)
+    assert description in res['description']
+
+
+def test_ddea_becomes_present(edap):
+    assert not edap.org_unit_exists("ddea")
+    edap.create_org_unit("ddea")
+    assert edap.org_unit_exists("ddea")
+
+
+def test_it_ddea_becomes_present(edap):
+    description = b'it'
+    assert not edap.object_exists_at(f"cn=it,{edap.DDEA_GROUP}", "posixGroup")
+    edap.create_ddea("it", display_name="it")
+    assert edap.object_exists_at(f"cn=it,{edap.DDEA_GROUP}", "posixGroup")
+    res = edap.get_ddea("it")
+    print("edap.get_ddea(it)", res)
+    assert description in res['description']
+
+
+def test_cdea_becomes_present(edap):
+    assert not edap.org_unit_exists("cdea")
+    edap.create_org_unit("cdea")
+    assert edap.org_unit_exists("cdea")
+
+
+def test_cz_cdea_becomes_present(edap):
+    description = b'cz'
```

Comment:
`b'cz'` is cz in "raw bytes" form, which is used to store things to LDAP. But we know that the description is a string and not e.g. a byte sequence representing an image. So the `get_cdea` method (and others) should return strings for string properties. Hint: You obtain string from bytes by [decoding bytes](https://stackoverflow.com/a/606199).

Reply:
For example in def test_it_division_becomes_present(edap): you use description = b"It division"
. Is it not the same as b'it_division' ? because there is not used decode:
def test_it_division_becomes_present(edap):
    description = b"It division"
    assert not edap.object_exists_at(f"cn=it,{edap.DIVISIONS_GROUP}", "posixGroup")
    edap.create_division("it", display_name=description)
    assert edap.object_exists_at(f"cn=it,{edap.DIVISIONS_GROUP}", "posixGroup")
    res = edap.get_division("it")
    assert description in res['description']

Reply:
I see, your code is consistent with the existing code. When I look at it now, I don't like it so much, but you did well on this one - consistency is the most important value.
So forget my comment, good job!

Reply:
Yes, first thing is that i look at existent code if there is already something and use it.